recipe/meta.yaml

```diff
@@ -1,32 +1,174 @@
+        - if not exist %PREFIX%\\Library\\lib\\pkgconfig\\libturbojpeg.pc exit 1  # [win]
+        - test -f ${PREFIX}/lib/pkgconfig/libjpeg.pc                         # [unix]
+        - if not exist %PREFIX%\\Library\\lib\\pkgconfig\\libjpeg.pc exit 1  # [win]
+        - test -f ${PREFIX}/lib/cmake/libjpeg-turbo/libjpeg-turboConfig.cmake   # [unix]
+        - test -f ${PREFIX}/lib/cmake/libjpeg-turbo/libjpeg-turboTargets.cmake  # [unix]
+        - if not exist %PREFIX%\\Library\\lib\\cmake\\libjpeg-turbo\\libjpeg-turboConfig.cmake exit 1   # [win]
+        - if not exist %PREFIX%\\Library\\lib\\cmake\\libjpeg-turbo\\libjpeg-turboTargets.cmake exit 1  # [win]
+
+  # Run exports package
+  - name: libturbojpeg{{ turbo_so_major_version }}
+    script: install.py
+    build:
+      run_exports:
+        - {{ pin_subpackage('libturbojpeg' ~ turbo_so_major_version) }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+      run:
+        - {{ pin_subpackage('_libturbojpeg_api', max_pin='x.x.x') }}          # [win]
+    test:
+      commands:
+      # Need these
+        - test -f ${PREFIX}/lib/libturbojpeg.so.{{ turbo_so_major_version }}        # [linux]
+        - test -f ${PREFIX}/lib/libturbojpeg.so.{{ turbo_so_version }}              # [linux]
+        - test -f ${PREFIX}/lib/libturbojpeg.{{ turbo_so_major_version }}.dylib     # [osx]
+        - test -f ${PREFIX}/lib/libturbojpeg.{{ turbo_so_version }}.dylib           # [osx]
+        - if not exist %PREFIX%\\Library\\bin\\turbojpeg.dll exit 1           # [win]
+  - name: libjpeg{{ jpeg_so_major_version }}
+    script: install.py
+    build:
+      run_exports:
+        - {{ pin_subpackage('libjpeg' ~ jpeg_so_major_version) }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+      run_constrained:
+        - jpeg <0.0.0a
+    test:
+      commands:
+      # Need these
+        - test -f ${PREFIX}/lib/libjpeg.so.{{ jpeg_so_major_version }}             # [linux]
+        - test -f ${PREFIX}/lib/libjpeg.so.{{ jpeg_so_version }}                   # [linux]
+        - test -f ${PREFIX}/lib/libjpeg.{{ jpeg_so_major_version }}.dylib          # [osx]
+        - test -f ${PREFIX}/lib/libjpeg.{{ jpeg_so_version }}.dylib                # [osx]
+        - if not exist %PREFIX%\\Library\\bin\\jpeg{{ jpeg_so_major_version }}.dll exit 1  # [win]
+
+  # Static library package
+  - name: libturbojpeg-static
+    script: install.py
+    test:
+      commands:
+      # Need these
+        - test -f ${PREFIX}/lib/libturbojpeg.a                                # [unix]
+        - if not exist %PREFIX%\\Library\\lib\\turbojpeg-static.lib exit 1    # [win]
+  - name: libjpeg-static
+    script: install.py
+    requirements:
+      run_constrained:
+        - jpeg <0.0.0a
+    test:
+      commands:
+      # Need these
+        - test -f ${PREFIX}/lib/libjpeg.a                                     # [unix]
+        - if not exist %PREFIX%\\Library\\lib\\jpeg-static.lib exit 1         # [win]
```

Comment:
As noted above, these outputs could be combined.

Reply:
Actually, we should probably drop/comment out the static outputs until someone complains they are missing.

Reply:
https://github.com/conda-forge/libjpeg-turbo-feedstock/pull/32#issuecomment-1972362932